### PR TITLE
Fixed the aggressive title replacements

### PR DESCRIPTION
**Pull request by sima-zhu**
Fixed the bug of `update-doc-metadata.ps1` script which replace the title too aggressively. 
Issue: https://github.com/Azure/azure-sdk/issues/1674

Tested in local with python storage pkg. 
image
image

### CODE REVIEW

**Review by scbedd — APPROVED** [review]
LGTM!

**Review by weshaggard — APPROVED** [review]
Lets wait until post release to merge.

**Comment by sima-zhu** [comment]
The change is pending for release cycle.

**Pull request merged**
